CHANGELOG.md

```diff
@@ -0,0 +1,91 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.3.0] - 2020-03-30
+
+### Added
+
+- Added `CHANGELOG.md`
+- Added `$_ZO_EXCLUDE_DIRS` to prevent certain directories from being added to the database.
+- Added support for generic POSIX shells.
+
+### Changed
+
+- Switched default database location to user's data directory.
+- Upgraded database schema to include a version number, and added automatic migration from `v0.2.x` databases (will be removed in `v0.4.0`).
+- Changed `migrate` subcommand to `import`.
+
+### Fixed
+
+- Used unique temporary database file names to achieve thread safety.
+- Fixed incomprehensive "could not allocate" message on database corruption.
```

Comment:
I'm not gonna go through every single entry, but the entries should be read like: `<Header>: <entry>`, e.g. `Fixed: Incomprehensive "could not allocate" message on database corruption.`

```suggestion
- Achieve thread safety by using a unique, temporary database file.
- Incomprehensive "could not allocate" message on database corruption.
```